### PR TITLE
Updated name filtering, reran pipeline

### PR DESCRIPTION
**Pull request by nrosed**
This PR covers the following conceptual updates: names to include 2-character first names, removing the guardian analysis, and storing all names excluded from the analysis.
Many files were changed, but this is because updating the name filter had many downstream effects  and most files edited are result files/images.

In this PR I need the following files to be reviewed:

1. utils/scraper_processing_utils.R 2 letter names are now permitted in the analysis
2. figure_notebooks/figure1.md guardian analysis is not longer included
3. figure_notebooks/figure2.Rmd and figure_notebooks/figure3.md guardian analysis is excluded and first authors are now included in the analysis
4. figure_notebooks/supp_tables.md guardian excluded
5. process_doi_data/process_speaker_fullnames.R now we need to keep track of all names excluded in the name-origin prediction analysis
6. utils/plotting_utils.R updated colors for plots

### CODE REVIEW

File: utils/scraper_processing_utils.R [hunk 1]
```diff
@@ -90,7 +90,7 @@ format_author_firstnames <- function(author_vec){
     author_vec = gsub("[.]", "", author_vec)
 
     # or less than 3 characters
```
Comment (danich1) [inline utils/scraper_processing_utils.R]:
```suggestion
    # or less than 2 characters
```
To keep the documentation consistent

File: figure_notebooks/figure1.md [hunk 2]
```diff
@@ -269,7 +269,7 @@ full_image <- image_append(image_scale(c(bottom_panel, news_nature_gg),3000),
 print(full_image)
```
Comment (danich1) [inline figure_notebooks/figure1.md:285]: According to your comments you removed the guardian from the analysis; however, the guardian category here still shows up in the figure. Is this intended?

**Review by danich1 — APPROVED** [review]
LGTM! Just one question and one suggestion.

File: figure_notebooks/figure1.md (hunk 2 above)
Comment (nrosed) [reply figure_notebooks/figure1.md:285]: oops! no, legend is fixed now.

Comment (nrosed) [reply figure_notebooks/figure1.md:285]: https://github.com/greenelab/nature_news_disparities/pull/34/commits/aa55aa31cce937cf99080c9b06caa574fa178d61

**Pull request merged**